tests/layer/test_watch_your_step.py

```diff
@@ -0,0 +1,102 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2020 Data61, CSIRO
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from stellargraph.layer import AttentiveWalk, WatchYourStep, get_embeddings
+import numpy as np
+from ..mapper.test_graphwave_generator import barbell
```

Comment:
Hmm should this import be updated
```suggestion
from ..test_utils.graphs import barbell
```